### ISSUE TITLE
When deleting a Subalbum reload should be the album above not the albums main page.

### ISSUE THREAD

**Issue by SerenaButler** [LycheeOrg/Lychee-Laravel#191]
Create an Album, create a subalbum, create more subalbums if you like (to get a deeper tree). Delete a Subalbum, you will find your self always on the albums main page but it "should" be the album "above" ..

**Comment by kamil4** [comment]
Yeah, I fixed this one in my local tree last night, along with several other front end issue. I'm hoping to submit the patches later today after I clean them up.

**Comment by SerenaButler** [comment]
Top! :-)
